scripts/local-cluster.sh

```diff
@@ -226,12 +226,12 @@ resolve() {
   if [[ -z "${action}" ]]; then
     log.usage
   fi
+  if [[ -z "${domain}" ]]; then
+    log.usage
+  fi
```

Comment:
This will require changes where it is used.

Reply:
Aware of any uses besides these?

https://github.com/elastisys/compliantkubernetes-apps/blob/d3ff4df7a14131991514b6d2fbdbb15e339850a2/.github/workflows/tests.yml#L93

https://github.com/elastisys/compliantkubernetes-apps/blob/d3ff4df7a14131991514b6d2fbdbb15e339850a2/.github/workflows/tests.yml#L139